docs/general/transform.md

```diff
@@ -0,0 +1,35 @@
+[Back to Guides](../README.md)
+
+# Transforms
+
+Transforms modify the casing of keys and keys referenced in values in serialized
+responses.
+
+Provided transforms:
+
+- `:camel` - ExampleKey
+- `:camel_lower` - exampleKey
+- `:dash` - example-key
+- `:unaltered` - the original, unaltered key
+- `nil` - use the adapter default
+
+Key translation precedence is as follows:
+
+##### SerializableResource option
+
+`transform` is provided as an option via render.
+
+```render json: posts, each_serializer: PostSerializer, transform: :camel_lower```
+
+##### Configuration option
+
+`transform` is set in `ActiveModelSerializers.config.transform`.
+
+```ActiveModelSerializers.config.transform = :camel_lower```
+
+##### Adapter default
+
+Each adapter has a default transform configured:
+
+- `Json` - `:unaltered`
+- `JsonApi` - `:dash`
```

Comment:
you can use a table here and above for better layout. see https://github.com/rails-api/active_model_serializers/tree/master/docs#integrations